bolides/bdf.py

```diff
@@ -927,43 +927,66 @@ def plot_dates(self, freq='1D', logscale=False,
         return fig, ax
 
     def add_website_data(self, ids=None):
-        """Pull light curve data from neo-bolide.ndc.nasa.gov.
+        """Pull light curve and integrated energy data from neo-bolide.ndc.nasa.gov.
 
         Downloads light curve data from neo-bolide.ndc.nasa.gov, placing it
         into the BolideDataFrame as a column of `~lightkurve.LightCurveCollection` objects.
-        The column is named ``'lightcurves'``
+        The column is named ``'lightcurves'``. Integrated energy data is also downloaded
+        for different satellites and placed into columns starting with ``'energy_'``.
+        This method will only work with BolideDataFrames having an ``_id`` column containing
+        IDs from GLM bolide detections at neo-bolide.ndc.nasa.gov.
 
         Parameters
         ----------
         ids : list of str
-            Optional list of strings representing the bolide ID's that light curves are wanted for.
-            If not used, a light curve is added for every bolide in the BolideDataFrame.
+            Optional list of strings representing the bolide ID's that additional data is
```

Comment:
In time, we will merge the IDs posted on the website and those in the pipeline. But for now, the `_id` is, I think, a MongoDB identifier.